src/main/java/lebron/task/TaskList.java

```diff
@@ -0,0 +1,75 @@
+package lebron.task;
+
+import lebron.Ui;
+import lebron.Lebron;
+
+import java.util.ArrayList;
+
+/**
+ * Represents the list of tasks.
+ *
+ * @author Nigel Tan
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+    private int position;
+    private Ui ui;
+
+    /**
+     * Constructor.
+     *
+     * @param lst The list of tasks.
+     */
+    public TaskList(ArrayList<Task> lst, Lebron lebron) {
+        this.tasks = lst;
+        this.ui = new Ui(lebron);
+        this.position = 1;
+    }
+
+    /**
+     * This method handles the add event.
+     *
+     * @param task the Task that the user wants to add
+     */
+    public String add(Task task) {
+        tasks.add(task);
+        String reply = ui.replyAdd(this.tasks, task);
+        return reply;
+    }
+
+    /**
+     * This method handles the markDone event when the user says done, marking the Task as done.
+     *
+     * @param pos the position of the Task in the ArrayList lst
```

Comment:
Shouldn't there be an @return statement in your javadoc since your method returns a value? I noticed the same issue in several other places too...